src/factorization.jl

```diff
@@ -503,30 +503,28 @@ end
 
 ## SparspakFactorization is here since it's MIT licensed, not GPL
 
-struct SparspakFactorization <: AbstractFactorization end
+Base.@kwdef struct SparspakFactorization <: AbstractFactorization
+    reuse_symbolic::Bool = true
+end
 
 function init_cacheval(::SparspakFactorization, A, b, u, Pl, Pr, maxiters::Int, abstol,
                        reltol,
                        verbose::Bool, assumptions::OperatorAssumptions)
     A = convert(AbstractMatrix, A)
-    p = Sparspak.Problem.Problem(size(A)...)
-    Sparspak.Problem.insparse!(p, A)
-    Sparspak.Problem.infullrhs!(p, b)
-    s = Sparspak.SparseSolver.SparseSolver(p)
-    return s
+    return sparspaklu(A)
```

Comment:
this performs an extra lu-factorization though? Can we avoid this and just build a blank/empty LU object to fill later?

Reply:
I see. Assuming I have all info on A this is possible but not in the API. I think I could  add a kwarg to sparspaklu.

Reply:
I think a kwarg would be a nice solution.